app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -526,6 +322,52 @@ class TopicLessonsFragmentTest {
     }
   }
 
+  private fun createTopicActivityIntent(internalProfileId: Int, topicId: String): Intent {
+    return TopicActivity.createTopicActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      internalProfileId,
+      topicId
+    )
+  }
+
+  private fun clickLessonTab() {
+    testCoroutineDispatchers.runCurrent()
+    onView(
+      allOf(
+        withText(TopicTab.getTabForPosition(1).name),
+        isDescendantOfA(withId(R.id.topic_tabs_container))
+      )
+    ).perform(click())
+    testCoroutineDispatchers.runCurrent()
+  }
+
+  private fun clickStoryItem(position: Int, targetViewId: Int) {
+    onView(
+      atPositionOnView(
+        R.id.story_summary_recycler_view,
+        position,
+        targetViewId
+      )
+    ).perform(click())
+  }
+
+  private fun scrollToPosition(position: Int) {
+    onView(withId(R.id.story_summary_recycler_view)).perform(
+      scrollToPosition<RecyclerView.ViewHolder>(
+        position
+      )
+    )
+  }
+
+  private fun matchStringOnStorySummaryListItem(position: Int, stringToMatch: String) {
```

Comment:
Ditto here per my comment in the other test file (prefer something like 'verify*...').

Reply:
done